modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/v10_5_0/ObjectFieldUpgradeProcess.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.internal.upgrade.v10_5_0;
+
+import com.liferay.petra.string.StringBundler;
+import com.liferay.portal.kernel.dao.jdbc.AutoBatchPreparedStatementUtil;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+
+/**
+ * @author Richard Jeremias
+ */
+public class ObjectFieldUpgradeProcess extends UpgradeProcess {
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		try (PreparedStatement preparedStatement1 = connection.prepareStatement(
+				StringBundler.concat(
+					"SELECT objectFieldId FROM ObjectField WHERE dbTableName ",
+					"= 'CPDefinition' AND name IN ('name','description',",
+					"'shortDescription')"));
+			PreparedStatement preparedStatement2 =
+				AutoBatchPreparedStatementUtil.concurrentAutoBatch(
+					connection,
+					"update ObjectField set dbTableName = ?, localized = ?  " +
+						"where objectFieldId = ?");
+			ResultSet resultSet = preparedStatement1.executeQuery()) {
+
+			while (resultSet.next()) {
+				preparedStatement2.setString(1, "CPDefinitionLocalization");
+				preparedStatement2.setBoolean(2, true);
+				preparedStatement2.setLong(
+					3, resultSet.getLong("objectFieldId"));
```

Comment:
we also don't need to pass variables because the values will be always the same:

`dbTableName = 'CPDefinitionLocalization', localized = [$TRUE$]`

Reply:
we don't need a class for this upgrade, see examples of `UpgradeProcessFactory.runSQL` in UpgradeStepRegistrator classes.

Thank you!